.github/workflows/sdk-integration.yml

```diff
@@ -1,52 +0,0 @@
-name: Integration tests
-
-on:
-  pull_request:
-
-jobs:
-  coverage:
-    name: Integration tests
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checkout sdk
-        uses: actions/checkout@v3
-        with:
-          path: 'sdk'
-      - name: Checkout nitro
-        uses: actions/checkout@v3
-        with:
-          repository: OffchainLabs/nitro
-          submodules: 'recursive'
-          path: 'nitro'
-          ref: 'test-node-prod'
-      - name: Setup node/yarn
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-          cache: 'yarn'
-          cache-dependency-path: '**/yarn.lock'
-      - name: Install dependencies
-        run: |
-          cd sdk
-          yarn
-          yarn build
-      - name: Start background nitro test-node.bash
-        run: |
-          cd nitro
-          ./test-node.bash --init --no-blockscout &
-      - name: Copy .env
-        run: |
-          cd sdk
-          cp ./.env-sample ./.env
-      - name: Wait for nitro startup
-        run: |
-          cd sdk/scripts
-          ./waitForNitro.sh
```

Comment:
we should probably delete this file?
https://github.com/OffchainLabs/arbitrum-sdk/blob/ci-speed-up/scripts/waitForNitro.sh
from its commit history it was added solely for ci purpose?
https://github.com/OffchainLabs/arbitrum-sdk/pull/119

Reply:
yep, it was only for ci